flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-values/src/test/java/org/apache/flink/cdc/connectors/values/source/ValuesDataSourceITCase.java

```diff
@@ -68,6 +68,7 @@ public void after() {
     private void executeDataStreamJob(ValuesDataSourceHelper.EventSetId type) throws Exception {
         StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
         env.enableCheckpointing(3000);
+        env.setParallelism(2);
```

Comment:
Just to speed up the running of `ValuesDataSourceITCase`, if you mind I can delete it or submit a separate PR.

Reply:
As reported in [FLINK-35127](https://issues.apache.org/jira/browse/FLINK-35127), using parallelism > 1 will crash this IT case, and get fixed in #3237 by removing  `HybridSource` entirely. Maybe this change needs further investigation.

Reply:
Glad to see it's fixed, I rollbacked the currently irrelevant commit.